### ISSUE TITLE
doesnot work in the two class classification model training

### ISSUE THREAD

**Issue by ailihong** [xialuxi/arcface-caffe#23]
hello guys,
   Thanks the author for his excellent work firstly.
   I use this arcface loss to finetune a classification model with two class,but I donot kown why it does not work.The loss does not decrease and accuracy is jumping.the parameter is m = 0.5, s=64.I have try some other parameters,but it is always same.
  Has anyone encountered this similar problem? thanks.
